pkg/apis/networking/types.go

```diff
@@ -48,31 +48,35 @@ const (
 
 // NetworkPolicySpec provides the specification of a NetworkPolicy
 type NetworkPolicySpec struct {
-	// Selects the pods to which this NetworkPolicy object applies. The array of
-	// ingress rules is applied to any pods selected by this field. Multiple network
-	// policies can select the same set of pods. In this case, the ingress rules for
-	// each are combined additively. This field is NOT optional and follows standard
+	// Selects the pods to which this NetworkPolicy object applies for ingress and/or egress
+	// (as determined by `policyTypes`).  The relevant one or ones of `ingress` and `egress`
+	// list connections that this policy allows to be made from and to the selected pods.
+	// Multiple NetworkPolicy objects can select a given pod, and the connections they
+	// allow are combined additively.
+	// If any NetworkPolicy applies to a given pod for ingress then (a) the default is
+	// to allow ingress to the pod only from the pod's node and (b) the applicable policies
+	// add allowed ingress to the pod; otherwise all ingress to the pod is allowed.
```

Comment:
> otherwise all ingress to the pod is allowed.

I originally read this wrong and thought you meant denied. I think maybe it would be clearer to spell it out like this:

> If any network policy applies to a given pod for ingress, then traffic is allowed from the pod's local node, traffic is allowed from the applicable policies' ingress rules, and all other ingress to the pod is denied. If no network policy selects a given pod, all ingress to that pod is allowed.

WDYT?